adafruit_rfm69.py

```diff
@@ -312,7 +312,7 @@ def __init__(  # pylint: disable=invalid-name
         self.reset()  # Reset the chip.
         # Check the version of the chip.
         version = self._read_u8(_REG_VERSION)
-        if version != 0x24:
+        if version not in [0x23, 0x24]:
```

Comment:
Will use a tuple here. Might be slightly more efficient.

```suggestion
        if version not in (0x23, 0x24):
```